v3/setup/secrets.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+declare -A SECRETMAP
+
+SECRETMAP[DATADOG_KEY]="/ddapikey"
+SECRETMAP[FD_GITHUB_CLIENT_ID]="/FD/GITHUB_CLIENT_ID"
+SECRETMAP[FD_GITHUB_CLIENT_SECRET]="/FD/GITHUB_CLIENT_SECRET"
+SECRETMAP[FD_GITHUB_ALLOWED_TEAMS]="/FD/GITHUB_ALLOWED_TEAMS"
+SECRETMAP[HUD_GITHUB_CLIENT_ID]="/HUD/client-id"
+SECRETMAP[HUD_GITHUB_CLIENT_SECRET]="/HUD/client-secret"
+SECRETMAP[MARATHON_USERNAME]="/marathon/username"
+SECRETMAP[MARATHON_PASSWORD]="/marathon/password"
+SECRETMAP[SUMOLOGIC_ACCESS_ID]="/sumologic_id"
+SECRETMAP[SUMOLOGIC_SECRET]="/sumologic_secret"
+SECRETMAP[SYSDIG_KEY]="/sysdigkey"
+
+TABLE=`sudo echo $SECRETS_TABLE`
```

Comment:
From this (now merged) PR in infrastructure repo: https://github.com/adobe-platform/infrastructure/commit/f71722bce135eafa17b81b53497bdbcab0891554